### PR TITLE
qbs 1.17.0

### PR DESCRIPTION
**Pull request by jobor**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?

-----

### CODE REVIEW

**Comment by bayandin** [comment]
```
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -stdlib=libc++ -headerpad_max_install_names  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -mmacosx-version-min=10.13 -Wl,-rpath,@executable_path/../Frameworks -single_module -dynamiclib -o libvisualstudiogenerator.dylib json.o msbuildfiltersproject.o msbuildqbsgenerateproject.o msbuildqbsproductproject.o msbuildsharedsolutionpropertiesproject.o msbuildsolutionpropertiesproject.o msbuildtargetproject.o visualstudiogenerator.o visualstudiogeneratorplugin.o visualstudioguidpool.o moc_msbuildfiltersproject.o moc_msbuildqbsgenerateproject.o moc_msbuildqbsproductproject.o moc_msbuildsharedsolutionpropertiesproject.o moc_msbuildsolutionpropertiesproject.o moc_msbuildtargetproject.o  -F/usr/local/Cellar/qt/5.15.0/lib -L/private/tmp/qbs-20200910-58139-pqaok6/qbs-src-1.17.0/src/plugins/generator/visualstudio/../../../../lib -lqbsmsbuild -L/private/tmp/qbs-20200910-58139-pqaok6/qbs-src-1.17.0/src/lib/msbuild/../../../lib -lqbscore -framework QtCore -framework DiskArbitration -framework IOKit    
ld: library not found for -lqbsmsbuild
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [../../../../lib/qbs/plugins/libvisualstudiogenerator.dylib] Error 1
make[2]: *** [sub-visualstudio-install_subtargets] Error 2
```

**Comment by ABBAPOH** [comment]
I cannot reproduce this locally, how do I debug this?

**Comment by SMillerDev** [comment]
> I cannot reproduce this locally, how do I debug this?

How are you trying to reproduce it locally?

**Comment by ABBAPOH** [comment]
> How are you trying to reproduce it locally?

$ which qmake
/Applications/Qt/5.15.0/clang_64/bin//quake
$ mkdir build && cd build
$ qmake -r ../qbs.pro CONFIG+=qbs_disable_rpath QBS_INSTALL_PREFIX=/usr/local
$ make install INSTALL_ROOT=/ -j12

builds and installs fine

**Comment by SMillerDev** [comment]
That foregoes almost all homebrew dependency management. What happens if you run `brew install --build-from-source` with your changes?

**Comment by ABBAPOH** [comment]
I can reproduce that with --build-from-source but I still cannot reproduce by calling the commands manually (using brew's qmake and the archive with sources). What else that could be different? Environment variables?

File: Formula/qbs.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Qbs < Formula
   depends_on "qt"
 
   def install
-    system "qmake", "qbs.pro", "-r", "QBS_INSTALL_PREFIX=#{prefix}", "CONFIG+=qbs_disable_rpath"
+    system "qmake", "qbs.pro", "QBS_INSTALL_PREFIX=#{prefix}", "CONFIG+=qbs_disable_rpath"
     system "make", "install", "INSTALL_ROOT=/"
```
Comment (ABBAPOH) [inline Formula/qbs.rb:19]: To fix the build you can call system "make" before calling "make install". It appears, qmake generates invalid dependencies for install targets.

Comment (jobor) [reply Formula/qbs.rb:20]: Thanks for investigating! I've adjusted qbs.rb accordingly.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @jobor!!

**Pull request closed without merging**